flow/embedded_views.h

```diff
@@ -9,23 +9,37 @@
 
 #include "flutter/fml/memory/ref_counted.h"
 #include "third_party/skia/include/core/SkCanvas.h"
+#include "third_party/skia/include/core/SkPath.h"
 #include "third_party/skia/include/core/SkPoint.h"
+#include "third_party/skia/include/core/SkRRect.h"
+#include "third_party/skia/include/core/SkRect.h"
 #include "third_party/skia/include/core/SkSize.h"
 
 namespace flutter {
 
+class FlutterEmbededViewTransformStack;
+class FlutterEmbededViewTransformElement;
+
 class EmbeddedViewParams {
  public:
   SkPoint offsetPixels;
   SkSize sizePoints;
+  std::shared_ptr<FlutterEmbededViewTransformStack> transformStack;
+
+  friend bool operator==(const EmbeddedViewParams& lhs,
```

Comment:
Ignore the operator== function here for now. This will be updated when I merged the other related PR into master and rebase this PR.